man/resolved.conf.xml

```diff
@@ -269,6 +269,22 @@
         in use.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DNSStubListenerExtra=</varname></term>
+        <listitem><para>Takes an IPv4 or IPv6 address to listen on. The address may optionally be prefixed by <literal>:</literal> and
+        a protocol name (<literal>udp</literal> or <literal>tcp</literal>). It may also be optionally suffixed by <literal>:</literal>
+        and a numeric port number. If the protocol is not specified, the service will listen on both IPv4 and IPv6. If the port is not
+        specified, then the service takes port as 53. This option may be used multiple times. Note that this is independent of the
```

Comment:
`This option may be ...` ->
`When an IPv6 address is specified with a port number, then the address must be in the square brackets. This option can be specified multiple times. If an empty string is assigned, then the all previous assignments are cleared.`

Reply:
thanks